docs/security.rst

```diff
@@ -0,0 +1,66 @@
+Security analysis
```

Comment:
Note : you need to add the page to the index file (`index.rst`) so that it show up in the main page and in menus